.tina/collections/employment.tsx

```diff
@@ -0,0 +1,154 @@
+import * as Schemas from "../../components/blocks";
+import { Collection } from "tinacms";
+import { seoSchema } from "../../components/util/seo";
+
+export const employmentSchema: Collection = {
+  label: "Employment Pages",
+  name: "employment",
+  format: "mdx",
+  path: "content/employment",
+  ui: {
+    router: ({ document }) => {
+      return "/employment";
+    },
+  },
+  fields: [
+    // @ts-ignore
+    seoSchema,
+    {
+      type: "object",
+      label: "Booking",
+      name: "booking",
+      fields: [
+        {
+          type: "string",
+          label: "Title",
+          name: "title",
+        },
+        {
+          type: "string",
+          label: "Subtitle",
+          name: "subTitle",
+        },
+        {
+          type: "string",
+          label: "Button Text",
+          name: "buttonText",
+        },
+        {
+          type: "image",
+          label: "Video Background",
+          name: "videoBackground",
+        },
+      ],
+    },
+    {
+      type: "rich-text",
+      label: "Body",
+      name: "_body",
+      templates: [...Schemas.pageBlocks],
+      isBody: true,
+    },
+    {
+      type: "rich-text",
+      label: "After body",
+      name: "afterBody",
+      templates: [...Schemas.pageBlocks],
+    },
```

Comment:
<img width="1802" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/38869720/e1108ec6-34e9-4496-a434-402348185a23">
I think this is supposed to go down under After benifits body to line up with the actual UI